infra/growth/campaigns/august.js

```diff
@@ -442,7 +442,13 @@ const augustConfig = {
             visible: true,
             nextLevelCondition: false,
             scope: 'campaign',
-            statusScope: 'user'
+            statusScope: 'user',
+            unlockConditionMsg: [
+              {
+                conditionTranslateKey: 'growth.attestation.requirement.telegram',
```

Comment:
We need to add this string to dapps/marketplace/src/constants/Fbt#enum.js then run "yarn run translate"

Reply:
Yes, Done.